@xen-orchestra/rest-api/src/rest-api/rest-api.type.mts

```diff
@@ -59,6 +60,7 @@ type XapiRecordByXapiXoRecord = {
   'VM-controller': XenApiVmWrapped
   'VM-snapshot': XenApiVmWrapped
   'VM-template': XenApiVmWrapped
+  repository: XenApiRepositoryWrapped
```

Comment:
Remove it.
`XapiRecordByXapiXoRecord` is reserved for XAPI objects to easily type `this.<XoObject>getXapiObject()`.

A backup repository is a pure XO object; therefore, it will not have an equivalent to a XAPI object.